docs/en/using-status/about-status-messages.md

```diff
@@ -0,0 +1,56 @@
+|:------------------------------|:---------------------|:---------------------|
+| End-to-end encryption         | :material-check-all: | :material-check-all: |
+| Network metadata encryption   | :material-check-all: | :octicons-x-24:      |
+| "Who talks to who" encryption | :material-check-all: | :material-check:     |
+| Censorship resistant          | :material-check-all: | :octicons-x-24:      |
+| Anonymous                     | :material-check-all: | :material-check:     |
+| Open source                   | :material-check-all: | :material-check:     |
+
+(:material-check-all: available, :material-check: available on some apps, :octicons-x-24: not available )
+
+## The basics
+
+- Your communications are end-to-end encrypted by default.
+- Your messages are not stored on centralized servers.
+- Only you and the message recipient can read your messages.
+- Status cannot identify you or other participants in the conversation.
+
+!!! tip
+    For answers to general questions about your Status messages, check the [Messages FAQs][messages-faqs] topic.
+
+## Understand peer-to-peer messaging
+
+Status provides resilient messaging without relying on centralized servers, data centers, or service providers. The Status app uses [Waku :octicons-tab-external-16:][waku]{:target="_blank"}, a peer-to-peer protocol for private, secure, and censorship-resistant communication.
+
+Unlike centralized networks (A), which depend on a central server to relay messages, peer-to-peer networks (B) encrypt and broadcast every message to all participant nodes. A node is a computer or smartphone that relays messages to other nodes or temporarily stores them for disconnected peers. Even when all nodes can access your messages, the content is encrypted so that only you and the intended recipients can read them.
+
+![The peer-to-peer network sends messages to every node and doesn't rely on a central server.](./about-status-messages/125-0-1-dark.png#only-light)
+![The peer-to-peer network sends messages to every node and doesn't rely on a central server.](./about-status-messages/125-0-1-light.png#only-dark)
+
+## Built-in privacy and security
+
+The Status app secures your messages the entire time they are in transit using end-to-end encryption. This industry standard protection uses strong encryption keys and ensures that only you and the recipients can read your messages. No one else (not even Status) can read your conversations or identify you or other participants.
+
+When you communicate with someone on the Status app, your messages, attachments, sender metadata, group chats, and group metadata are all end-to-end encrypted.
+
+Status incorporates the [Perfect Forward Secrecy :octicons-tab-external-16:][perfect-foward-secrecy]{:target="_blank"} (PFS) encryption mechanism, ensuring that encryption keys change on every message. If your keys are compromised, only the associated message is compromised. All previous and future messages remain private.
```

Comment:
Is that true cc @richard-ramos ?

Reply:
I believe @cammellos knows about this and can (hopefully) have a quick look at this content 🙏

Reply:
That's correct, 
though not completely accurate

```
All previous and future messages remain private.
```

PFS covers only previous messages, future messages are not covered by PFS, that's a different property (post compromise security/backward privacy), and what we use has a good post compromise security, but it's not watertight.

Reply:
Thanks, @cammellos!